### PR TITLE
[Concept Entry] JavaScript Window

### PR DESCRIPTION
**Pull request by SSwiniarski**
### Description

Concept entry for JavaScript `window` object.

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/javascript/concepts/window/window.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+Tags:
+  - 'Browsers'
+  - 'DOM'
+  - 'Objects'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `window` object is the global object of JavaScript when run within a browser and represents the browser window. It is supported by all browsers. It contains as members all global JavaScript objects, functions and variables. Such variables become properties of the `window` object, such functions become methods of the `window` object.
```
Comment (KTom101) [inline content/javascript/concepts/window/window.md:16]:
```suggestion
The **`window`** object is the global object of JavaScript when run within a browser and represents the browser window. All browsers support it. It contains as members all global JavaScript objects, functions, and variables. Such variables become properties of the `window` object, such functions become methods of the `window` object.
```

File: content/javascript/concepts/window/window.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+> **Note:** If a browser window contains [`<iframe>`](https://www.codecademy.com/resources/docs/html/elements/iframe) tags, there will be one `window` object for the page containing the `<iframe>` tags, and an additional `window` object for the page in each `<iframe>`
```
Comment (KTom101) [inline content/javascript/concepts/window/window.md:18]:
```suggestion
> **Note:** If a browser window contains [`<iframe>`](https://www.codecademy.com/resources/docs/html/elements/iframe) tags, there will be one `window` object for the page containing the `<iframe>` tags, and an additional `window` object for the page in each `<iframe>`.
```

**Review by KTom101 — CHANGES_REQUESTED** [review]
@SSwiniarski I added some minor changes

**Comment by SSwiniarski** [comment]
@KTom101, I updated with your suggestions.

**Review by KTom101 — APPROVED** [review]
I'll pass this on to a second review.

**Review by caupolicandiaz — APPROVED** [review]
@SSwiniarski @KTom101 LGTM

**Comment by yangc95** [comment]
🙌 @SSwiniarski live link: https://www.codecademy.com/resources/docs/javascript/window

Screenshot 2023-05-29 at 7 26 34 PM

cc: @KTom101

**Pull request merged**